src/geometry/euclidean/oriented_angle.lean

```diff
@@ -245,79 +244,79 @@ end
-  (2 : ℤ) • hb.oangle x (r • x) = 0 :=
+  (2 : ℤ) • b.oangle x (r • x) = 0 :=
 begin
   rcases lt_or_le r 0 with (h|h);
     simp [h]
 end
 
 /-- Twice the angle between two multiples of a vector is 0. -/
 @[simp] lemma two_zsmul_oangle_smul_smul_self (x : V) {r₁ r₂ : ℝ} :
-  (2 : ℤ) • hb.oangle (r₁ • x) (r₂ • x) = 0 :=
+  (2 : ℤ) • b.oangle (r₁ • x) (r₂ • x) = 0 :=
 begin
   by_cases h : r₁ = 0;
     simp [h]
 end
 
 /-- The oriented angle between two vectors is zero if and only if the angle with the vectors
 swapped is zero. -/
-lemma oangle_eq_zero_iff_oangle_rev_eq_zero {x y : V} : hb.oangle x y = 0 ↔ hb.oangle y x = 0 :=
+lemma oangle_eq_zero_iff_oangle_rev_eq_zero {x y : V} : b.oangle x y = 0 ↔ b.oangle y x = 0 :=
 by rw [oangle_rev, neg_eq_zero]
 
 /-- The oriented angle between two vectors is zero if and only if they are on the same ray. -/
-lemma oangle_eq_zero_iff_same_ray {x y : V} : hb.oangle x y = 0 ↔ same_ray ℝ x y :=
+lemma oangle_eq_zero_iff_same_ray {x y : V} : b.oangle x y = 0 ↔ same_ray ℝ x y :=
 by rw [oangle, complex.arg_coe_angle_eq_iff_eq_to_real, real.angle.to_real_zero,
        ←complex.same_ray_iff_arg_div_eq_zero, ←linear_isometry_equiv.coe_to_linear_equiv,
        same_ray_map_iff, same_ray_comm]
 
 /-- The oriented angle between two vectors is `π` if and only if the angle with the vectors
 swapped is `π`. -/
-lemma oangle_eq_pi_iff_oangle_rev_eq_pi {x y : V} : hb.oangle x y = π ↔ hb.oangle y x = π :=
+lemma oangle_eq_pi_iff_oangle_rev_eq_pi {x y : V} : b.oangle x y = π ↔ b.oangle y x = π :=
 by rw [oangle_rev, neg_eq_iff_neg_eq, eq_comm, real.angle.neg_coe_pi]
 
 /-- The oriented angle between two vectors is `π` if and only they are nonzero and the first is
 on the same ray as the negation of the second. -/
 lemma oangle_eq_pi_iff_same_ray_neg {x y : V} :
-  hb.oangle x y = π ↔ x ≠ 0 ∧ y ≠ 0 ∧ same_ray ℝ x (-y) :=
+  b.oangle x y = π ↔ x ≠ 0 ∧ y ≠ 0 ∧ same_ray ℝ x (-y) :=
 begin
-  rw [←hb.oangle_eq_zero_iff_same_ray],
+  rw [←b.oangle_eq_zero_iff_same_ray],
   split,
   { intro h,
     by_cases hx : x = 0, { simpa [hx, real.angle.pi_ne_zero.symm] using h },
     by_cases hy : y = 0, { simpa [hy, real.angle.pi_ne_zero.symm] using h },
     refine ⟨hx, hy, _⟩,
-    rw [hb.oangle_neg_right hx hy, h, real.angle.coe_pi_add_coe_pi] },
+    rw [b.oangle_neg_right hx hy, h, real.angle.coe_pi_add_coe_pi] },
   { rintro ⟨hx, hy, h⟩,
-    rwa [hb.oangle_neg_right hx hy, ←real.angle.sub_coe_pi_eq_add_coe_pi, sub_eq_zero] at h }
+    rwa [b.oangle_neg_right hx hy, ←real.angle.sub_coe_pi_eq_add_coe_pi, sub_eq_zero] at h }
 end
 
 /-- The oriented angle between two vectors is zero or `π` if and only if those two vectors are
 not linearly independent. -/
 lemma oangle_eq_zero_or_eq_pi_iff_not_linear_independent {x y : V} :
-  (hb.oangle x y = 0 ∨ hb.oangle x y = π) ↔ ¬ _root_.linear_independent ℝ ![x, y] :=
+  (b.oangle x y = 0 ∨ b.oangle x y = π) ↔ ¬ _root_.linear_independent ℝ ![x, y] :=
```

Comment:
The use of `_root_.` here was because of a conflict with the `orthonormal.linear_independent` lemma. Now that this is in the `orthonormal_basis` namespace instead of the `orthonormal` one, `_root_.` should no longer be needed here.

Reply:
Changed, thanks!